arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -291,7 +291,8 @@
               "enum": [
                 "Swarm",
                 "DCOS",
-                "Custom"
+                "Custom",
+                "Kubernetes"
```

Comment:
I see. In that case it would be ok. If the sdk is in preview then we would bump the minor version else we bump the major version and follow semantic versioning.